front/src/UI/organisms/AlarmChat/index.tsx

```diff
@@ -71,11 +115,55 @@ export const AlarmChat = () => {
       </h2>
 
       <AccordionPanel pb={4} bg={ALARM_BACKGROUND_COLOR}>
-        <Flex flexDirection="column">
-          {chatLog.map(({ index, type, chatId, message, createdAt }) => (
-            <AlarmChatMessage key={index} type={type} chatID={chatId} message={message} createdAt={createdAt} />
-          ))}
+        <Flex
+          flexDirection="column"
+          overflowX="hidden"
+          overflowY="auto"
+          height="258px"
+          style={{ scrollbarWidth: 'thin' }}
+          css={{
+            '&::-webkit-scrollbar': {
+              width: '5px',
+            },
+            '&::-webkit-scrollbar-track': {
+              width: '5px',
+              background: 'lightgray',
+            },
+            '&::-webkit-scrollbar-thumb': {
+              background: 'gray',
+              borderRadius: '24px',
+            },
+          }}
+          ref={scrollRef}
+        >
+          <AlarmChatMessagesBox
+            chatLog={data.chat.chatLog.map((item) => {
+              return {
+                ...item,
+                createdAt: compareTimeLapseToString(new Date(), postgresTimeToDate(item.createdAt)),
+              };
+            })}
+            chatIndex={data.chat.index}
+            subscribeToNewMessage={() =>
+              subscribeToMore({
+                document: CHATLOG_SUBSCRIPTION,
+                variables: { uuid: currentChat },
+                updateQuery: (prev, { subscriptionData }) => {
+                  if (!subscriptionData.data) return prev;
```

Comment:
```suggestion
                  if (!subscriptionData.data) {return prev;}
```

{}는 꼭 한줄이여도 넣는게 가독성에서 좋다고 생각하빈다.